kie-api/src/main/java/org/kie/api/builder/KieBuilder.java

```diff
@@ -39,6 +41,13 @@
     KieBuilder buildAll();
 
     /**
+     * Builds all the KieBases contained in the KieModule for which this KieBuilder has been created
+     * @param classFilters Used to prevent compilation of Java source files.
+     *          A Java source file vetoed by any of the given filters will not be compiled.
+     */
+    KieBuilder buildAll(Collection<KieClassFilter> classFilters);
```

Comment:
Rethinking to what suggested by @psiroky I'm even not sure that this feature is general enough to be added to our public API.

@mbarkley I suggest to move the signature of that method to our internal API https://github.com/droolsjbpm/droolsjbpm-knowledge/blob/master/kie-internal/src/main/java/org/kie/internal/builder/InternalKieBuilder.java so you could just downcast to the InternalKieBuilder and use that feature without exposing it to public API, at least until we don't have a better understanding. We are always in time to promote this method to public API if we will find that it can be useful in other situations.